CefSharp.Core/Internals/CefRegisterCdmCallbackWrapper.h

```diff
@@ -0,0 +1,65 @@
+// Copyright � 2010-2016 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#pragma once
+
+#include "Stdafx.h"
+#include "CefWrapper.h"
+
+namespace CefSharp
+{
+    namespace Internals
+    {
+        private class CefRegisterCdmCallbackWrapper : public CefRegisterCdmCallback
+        {
+        private:
+            gcroot<TaskCompletionSource<CdmRegistration^>^> _taskCompletionSource;
```

Comment:
Do you want a base concrete implementation of IRegisterCdmCallback as well or just implement an example within the example project?